### ISSUE TITLE
Admin Services API for View Services

### ISSUE THREAD

**Issue by grahamwallis** [odpi/egeria#3836]
I would have assigned this issue to myself, Mandy and David but for some reason I seem to be able to either assign it to myself (only) or to Mandy and David (only). I am happy to 'own' it, but it needs a conversation first, so assigning to Mandy and David so it gets their attention.

I have been extending the Admin Guide for the administration of view services and noticed the following:

1. There is no API for retrieving the configuration of an individual view service. This seems like an obvious operation to include and should have the following REST interface:
```
GET {serverURLRoot}/open-metadata/admin-services/users/{adminUserId}/servers/{serverName}/view-services/{serviceURLMarker}
```
The response should be a ViewServiceConfigResponse containing a ViewServiceConfig object.

2. There is no API for clearing an individual view service. Again, this seems like an obvious operation to include and should have the following REST interface:
```
DELETE {serverURLRoot}/open-metadata/admin-services/users/{adminUserId}/servers/{serverName}/view-services/{serviceURLMarker}
```

3. There is an API for enabling all configured view services. However, it currently is coded to accept a RequestBody consisting of a single ViewServiceConfig. This is analogous to how the corresponding operation is designed in the Access Services Admin API, which accepts a map of accessServiceOptions (which may or may not be sufficient, but probably doesn't matter as I think the access services will be enabled with default settings). However, for 'enable all' operation for view services, it does not seem to me that it's possible to adopt default values. Each Solution View Service will require an `omagServerName` and `omagServerPlatformURL` which *could* be defaulted, but it seems very likely that different OMASs would be on different platforms/servers so this seems an odd approach. Furthermore, each Integration View Service will require a set of resource endpoints for which there is no practical default value, other than to make it empty, which would render view services that are not usable. In my opinion I think this operation should probably be removed from the View Services Admin API.

I'd appreciate comments and discussion on the above points. In the description in the Admin Guide I have, for the time being, assumed that the first two operations will be added, and that the third will be removed. If we decide that this is the most desirable outcome I am happy to make the changes to the API. If not I will change the descriptions in the documentation, which I will not check in yet.

**Comment by davidradl** [comment]
Point 1 looks useful, though not in keeping with the style of the other admin calls, which do not return what has been created or updated in their responses.    
Point 2 seems reasonable.
Point 3 seems related to https://github.com/odpi/egeria/issues/3662 - apart from my mistake around starting when I meant configuring.  I think having configure all governance solutions view service call and a configure all integration view services could make sense especially for demo or testing purposes, but are unlikely to be used in production. I think the existing configure all view services API does not make sense any more; I am happy for this to be deprecated / removed. .

**Comment by grahamwallis** [comment]
Point 1: This is for a GET operation, so I would expect it to return the configuration.

Point 3: Splitting them out as you suggest may make sense. It may be useful to be able to start all solution view services, provided you are happy to adopt the same omagServerName and omagServerPlatformURL across all of them. For integration view services it would similar - i.e. we would make them all adopt the same set of resource endpoints. I'm not sure whether these 'one-szie-fits-all' approaches are suitable for solution view services or integration view services, but if they are then we could split them as you suggested.

**Comment by davidradl** [comment]
@grahamwallis  sorry misread point 1. The get seems a good addition as well.

**Comment by grahamwallis** [comment]
Fixed by #3848
